### ISSUE TITLE
Wallet does not show secret key

### ISSUE THREAD

**Issue by christopherj525** [keybase/keybase-issues#4213]
When I go to the wallet, it tells me "Stellar Transactions are No Longer Supported in the Keybase App" Followed by the account ID, Balance, and instead of the Secret Key it says "You can only view your secret key on mobile devices because this is a mobile-only account"

It says this on both my laptop and my mobile device running MacOS and iOS respectively.

**Comment by GwynethLlewelyn** [comment]
Interesting. On macOS, I'm running Version 6.2.4-20231019203304+5cfcf6b41e — the latest on the prerelease page — and I have no issues viewing my Secret Key there. The same on iOS (where the latest version on the App Store is 6.2.3-20230927213312+eba57c1814).

What happens when you do it from the command line (i.e., `keybase wallet export <insert your Stellar id here>`)? I have no issues with that, either.

**Comment by christopherj525** [comment]
Keybase GUI Version: 6.2.3-20230928143931+eba57c1814

` % keybase wallet export <Stellar id>
Please enter the Keybase password for <user> (8+ characters) to export Stellar secret keys: 
▶ ERROR not a mobile device (code 3110)`

**Comment by GwynethLlewelyn** [comment]
Ok, clearly whatever setup you have, it's different from mine.

I'm also assuming that you haven't enabled Lockdown mode, right?

I wonder...  when you registered with Keybase, did you use your mobile phone first, or the Mac?

One (remote) possibility is that somehow Keybase has registered/recorded the mobile phone with the iOS version _at the time you registered_ and for some strange reason is now unable to "remember" who you are... because the login/password _seems_ to come from a completely different device.

**Comment by christopherj525** [comment]
No, not in lockdown. I registered in October of 2014, so I think there was only desktop client at the time. Memory from that far past on this topic is not solid though.

**Comment by GwynethLlewelyn** [comment]
I think you're right... sorry, I ran out of ideas. Interestingly enough, I'd expect the _reverse_ to happen, i.e. Keybase complaining that it couldn't export the keys on a _mobile_ device, because you had registered it on the _desktop client_ before. But clearly that's not what's happening.

**Comment by lostchopstik** [comment]
Chiming in with same issue but on Android/PC. 

Command line on PC says: `- ERROR not a mobile device (code 3110)`

Phone says (from what I can see because the message goes off-screen): `You can only view your secret key on [...] devices because this is a mobile-only a...`

Update: I have a spare iOS device so I logged in with that and still got the same cut off message as my Android. I don't know if this is a bug or if they will only show the key to my original devices which I have but had signed out of or uninstalled the app. Very annoying!

**Comment by naoisegolden** [comment]
I had the same issue in the mobile app, it was solved by simply waiting a week. 

As mentioned, wallet secret key is only accessible by mobile app. But Keybase puts a 1-week wait for new registered mobile apps before being able to access this info. This fact is not mentioned in the error message. I found a mention about the 1-week wait by chance in the documentation, but I can't find it now to link it.

**Comment by lostchopstik** [comment]
> I had the same issue in the mobile app, it was solved by simply waiting a week.
> 
> As mentioned, wallet secret key is only accessible by mobile app. But Keybase puts a 1-week wait for new registered mobile apps before being able to access this info. This fact is not mentioned in the error message. I found a mention about the 1-week wait by chance in the documentation, but I can't find it now to link it.

Nice! Thanks, I'll wait around and see what happens.

**Comment by loxy** [comment]
For anybody else running into this issue. You need to re-install your client. See also: https://github.com/keybase/keybase-issues/issues/4219
